content/departments/engineering/dev/tools/grpc_cs_enablement.md

```diff
@@ -0,0 +1,165 @@
+# gRPC CS Enablement Guide / Primer
+
+## What is gRPC?
+
+[gRPC](https://grpc.io/) is a high-performance [Remote Procedure Call (RPC)](https://en.wikipedia.org/wiki/Remote_procedure_call) framework developed by Google.
+
+### Sidebar: What is an RPC (Remote Procedure Call)?
+
+At its most basic level, an [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) allows one program to cause a function to execute in another program (commonly on another physical machine).
+
+In simpler terms, it's a way for different parts of a system, which could be located on different machines, to talk to each other and perform tasks.
+
+Imagine being able to “press a button in one” application to get information or perform an action in another application located somewhere else; that's the essence of an [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call).
+
+Here is a pseudo-code example of what an [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) can look like:
+
+```javascript
+// On Server Machine
+function add(a, b) {
+  return a + b
+}
+
+/// ---------------
+
+// On Client Machine
+result = remoteCall('add', 5, 3) // Makes an RPC to the 'add' function on the server
+print(result) // Output will be 8
+```
+
+## What is changing?
+
+Starting in Sourcegraph `5.2.X` , we are switching to use gRPC for Sourcegraph’s **_internal communication (service to service only)_** by default.
```

Comment:
How about making it concrete here?

```suggestion
Starting in Sourcegraph `5.2.X` , we are switching to use gRPC for Sourcegraph’s **_internal communication (service to service only)_** by default.

In layman's terms: instead of JSON being sent via HTTP between `repo-updater` and `gitserver`, both services now use gRPC to talk to each other.
```